services/hapi/src/index.js

```diff
@@ -0,0 +1,49 @@
+'use strict'
+
+const updateBpStats = require('./libs/sync-bp-stats')
+const updateUserRatings = require('./libs/sync-user-rating')
+
+const Hapi = require('@hapi/hapi')
+
+const init = async () => {
+  const server = Hapi.server({
+    port: 3005,
+    host: '0.0.0.0'
+  })
+
+  server.route({
```

Comment:
@rubenabix  We are not using the root path , should we remove it?

Reply:
@xavier506 Yes, we should remove it.